system/ui/setup.py

```diff
@@ -299,6 +311,18 @@ def handle_keyboard_result(result):
     self.keyboard.set_title("Enter URL", "for Custom Software")
     gui_app.set_modal_overlay(self.keyboard, callback=handle_keyboard_result)
 
+  def use_openpilot(self):
+    if os.path.isdir(OPENPILOT_CACHE) and get_short_branch(OPENPILOT_CACHE) == "release3-staging":
+      shutil.copyfile(INSTALLER_SOURCE, INSTALLER_DESTINATION)
```

Comment:
can the installer be (zipapp'd) python? then we can just import the same functionality here which would be nice